graylog2-web-interface/src/components/graylog/Row.tsx

```diff
@@ -18,10 +18,12 @@
 import { Row as BootstrapRow } from 'react-bootstrap';
 import styled, { css } from 'styled-components';
 
+export const BOTTOM_MARGIN = 6;
```

Comment:
Is this something we might want to centralize, e.g. by moving it to the theme?

Reply:
I like the idea and created a new section in the theme for these margins, in a separate PR: 
https://github.com/Graylog2/graylog2-server/pull/10676